Revert "change to 8240 in to handle high multiplicity bug"

silviodonato opened the pull request:
Reverts cms-externals/pythia8#22

Generators required to have two versions of 10_6_14.
- CMSSW_10_6_14 -> without https://github.com/cms-sw/cmsdist/pull/5949
- CMSSW_10_6_14_Pyt8240BugFix -> with https://github.com/cms-sw/cmsdist/pull/5949

Given that https://github.com/cms-sw/cmsdist/pull/5949 was already merged,
this is my plan
- build CMSSW_10_6_14_Pyt8240BugFix (done https://github.com/cms-sw/cmssw/issues/30540)
- revert https://github.com/cms-sw/cmsdist/pull/5949
- build CMSSW_10_6_14

In order to revert https://github.com/cms-sw/cmsdist/pull/5949 we need to create a new tag because another  #24 was merged in the middle
cc: @alberto-sanchez @agrohsje @efeyazgan @mkirsano @qliphy @SiewYan

silviodonato committed:
Revert "change to 8240 in to handle high multiplicity bug"

cmsbuild commented:
A new Pull Request was created by  @silviodonato (Silvio Donato) for branch cms/240.

@cmsbuild, @smuzaffar, @mrodozov, @tulamor can you please review it and eventually sign? Thanks.
cms-bot commands are listed <a href="http://cms-sw.github.io/cms-bot-cmssw-cmds.html">here</a>

silviodonato commented:
please test for CMSSW_10_6_X

cmsbuild commented:
The tests are being triggered in jenkins.
Test Parameters:
  - **RELEASE_FORMAT** = CMSSW_10_6_X
- CMSSW_10_6_X_2020-07-04-1100/slc7_amd64_gcc700: https://cmssdt.cern.ch/jenkins/job/ib-run-pr-tests/7696/console Started: 2020/07/04 22:52

mrodozov commented:
I think there was some issue with the bot for 10_6 , lets hope not

cmsbuild commented:
+1
Tested at: 4311abdadcdfd2eeb2c1dcb72a0de978d4180b97
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-d88fc4/7696/summary.html
CMSSW: CMSSW_10_6_X_2020-07-04-1100
SCRAM_ARCH: slc7_amd64_gcc700

cmsbuild commented:
Comparison job queued.

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-d88fc4/7696/summary.html

Comparison Summary:
 - No significant changes to the logs found
 - Reco comparison results: 41212 differences found in the comparisons
 - DQMHistoTests: Total files compared: 34
 - DQMHistoTests: Total histograms compared: 3214558
 - DQMHistoTests: Total failures: 417099
 - DQMHistoTests: Total nulls: 3
 - DQMHistoTests: Total successes: 2797122
 - DQMHistoTests: Total skipped: 334
 - DQMHistoTests: Total Missing objects: 0
 - DQMHistoSizes: Histogram memory added: 24.974 KiB( 33 files compared)
 - DQMHistoSizes: changed ( 10224.0 ): -0.786 KiB       SiStrip/MechanicalView
 - DQMHistoSizes: changed ( 10824.0,... ): 12.992 KiB       HLT/HLTEgammaValidation
 - DQMHistoSizes: changed ( 250202.181 ): -0.225 KiB       SiStrip/MechanicalView
 - Checked 140 log files, 14 edm output root files, 34 DQM output files

silviodonato commented:
> I think there was some issue with the bot for 10_6 , lets hope not

@mrodozov 

> Comparison is ready
> https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-d88fc4/7696/summary.html
> 
> Comparison Summary:
> 
>     * No significant changes to the logs found
> 
>     * Reco comparison results: 41212 differences found in the comparisons
> 
>     * DQMHistoTests: Total files compared: 34
> 
>     * DQMHistoTests: Total histograms compared: 3214558
> 
>     * DQMHistoTests: Total failures: 417099
> 
>     * DQMHistoTests: Total nulls: 3
> 
>     * DQMHistoTests: Total successes: 2797122
> 
>     * DQMHistoTests: Total skipped: 334
> 
>     * DQMHistoTests: Total Missing objects: 0
> 
>     * DQMHistoSizes: Histogram memory added: 24.974 KiB( 33 files compared)
> 
>     * DQMHistoSizes: changed ( 10224.0 ): -0.786 KiB       SiStrip/MechanicalView
> 
>     * DQMHistoSizes: changed ( 10824.0,... ): 12.992 KiB       HLT/HLTEgammaValidation
> 
>     * DQMHistoSizes: changed ( 250202.181 ): -0.225 KiB       SiStrip/MechanicalView
> 
>     * Checked 140 log files, 14 edm output root files, 34 DQM output files

As expected the comparison result is the opposite of 
https://github.com/cms-externals/pythia8/pull/22#issuecomment-642009069

silviodonato commented:
@smuzaffar can I merge this PR ? do you have other suggestions about how to build CMSSW_10_6_14 (ie. current CMSSW_10_6_X without  cms-sw/cmsdist#5949)

smuzaffar commented:
So we do not need this patch? If yes then yes we shoudl merge it and update cmsdist to use the correct commit in 10.6.X

smuzaffar commented:
+externals

cmsbuild commented:
This pull request is fully signed and it will be integrated in one of the next cms/240 IBs (tests are also fine). This pull request will now be reviewed by the release team before it's merged. @silviodonato, @dpiparo (and backports should be raised in the release meeting by the corresponding L2)

silviodonato commented:
merge
@smuzaffar PdmV is going to make a validation CMSSW_10_6_14 vs CMSSW_10_6_14_Pyt8240BugFix in order to decide about this bugfix
